code/modules/projectiles/guns/ballistic.dm

```diff
@@ -475,6 +476,13 @@
 		if(misfire_probability > 0)
 			. += "<span class='danger'>Given the state of the gun, there is a [misfire_probability]% chance it'll misfire.</span>"
 
+/// Creates a new mag object to draw bullet data from, since you cannot draw data from something that doesn't exist
+/obj/item/gun/ballistic/proc/get_notes()
+	if(!mag_type)
+		return
+	var/obj/item/ammo_box/magazine/exam_mag = new src.mag_type
+	return uppertext("AMMO: [exam_mag.caliber]")
+
```

Comment:
That way you don't have to do `offensive_notes += get_notes()` for each new place you want to add info to the string